scrapi/migrations.py

```diff
@@ -19,97 +14,87 @@
 @tasks.task_autoretry(default_retry_delay=30, max_retries=5)
 def rename(docs, target=None, **kwargs):
     assert target, "To run this migration you need a target."
+
     for doc in docs:
-        raw = RawDocument({
-            'doc': doc.doc,
-            'docID': doc.docID,
-            'source': target,
-            'filetype': doc.filetype,
-            'timestamps': doc.timestamps,
-            'versions': doc.versions
-        })
+        raw_doc = doc.raw
```

Comment:
`new_doc = copy.copy(raw_doc.attributes)`? Also might want to do a `deepcopy` here.